src/components/Login/LoginContainer.tsx

```diff
@@ -191,6 +191,9 @@ export const InitialisationContainer: FunctionComponent<NavigationProps> = ({
       try {
         const { key, endpoint } = decodeQr(qrCode);
         Vibration.vibrate(50);
+        const endpointCheck = process.env.DOMAIN_CHECK as string;
```

Comment:
Another way to approach this is to set this as an `extra` field in `app.config.ts`, this provides the ability to check if it exists before continuing. So if someone were to try building the app without providing this prop, an error will show.

Reply:
Good point, I'll make the change haha

Reply:
Added!